pandas/tests/arrays/interval/test_ops.py

```diff
@@ -4,8 +4,12 @@
 
 from pandas import Interval, IntervalIndex, Timedelta, Timestamp
 from pandas.core.arrays import IntervalArray
+from pandas.tests.scalar.interval.conftest import overlaps_cases
 import pandas.util.testing as tm
 
+# silence "imported but unused" warning by flake8
+overlaps_cases
+
```

Comment:
you are missing the point, you need to move the conftest to a higher level to avoid this import smell

Reply:
Shit. Here we go again.

@jreback , this is the second time you've told me "I'm missing the point", assuming I'm a moron, and been monumentally dead wrong in both times and on both counts. It's also the second PR I'm going to abandon because of your patronizing and offensive manner. 

I've asked for you to stay away from my PRs and leave review to saner team members. But here we are again, with you talking shit, and me not willing to take it.

**There is no sane place to put the conftest**  because of the way the test hierarchy is built.
To share a conftest between `pandas/tests/arrays/interval` and `pandas/tests/scalar/interval`
It would have to be placed in `pandas/tests` and then be loaded by dozens of unrelated modules. That's far worse than these silly two lines. Buy for you its obvious that other developers couldn't possibly have any judgment, and you have have to micromanage them to the point of disgust. 

The hell with this.

@pandas-dev Your project has a problem. jreback should not be allowed near contributors.